app/main.py

```
@@ -0,0 +1,327 @@
+    return {
+        "service": "memOS.as",
+        "status": "running",
+        "description": "Memory and tool discovery hub for AI agents"
+    }
+
+@app.get("/health")
+async def health_check():
+    """Detailed health check with service status"""
+    try:
+        # Check PostgreSQL connection
+        postgres_client = get_postgres_client()
```

Ruff (0.12.2)

Local variable `postgres_client` is assigned to but never used

Remove assignment to unused variable `postgres_client`

(F841)